apps/cowswap-frontend/src/modules/swap/pure/warnings.tsx

```diff
@@ -40,9 +41,11 @@ export const SwapWarningsTop = React.memo(function (props: SwapWarningsTopProps)
     tradeUrlParams,
     isNativeSellInHooksStore,
   } = props
+  const sellToken = trade?.inputAmount.currency
 
   return (
     <>
+      {sellToken && !isNativeSellInHooksStore && <MetamaskTransactionWarning sellToken={sellToken} />}
```

Comment:
Why `isNativeSellInHooksStore`?

Reply:
We don't need to display this warning in Hooks store, because eth-flow is disabled there

Reply:
But then, it would also never be displayed since eth can't be sold there, no?